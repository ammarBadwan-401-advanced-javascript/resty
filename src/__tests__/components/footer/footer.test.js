import React from 'react';
import { shallow, mount} from 'enzyme';
import renderer from 'react-test-renderer';

import Footer from '../../../components/footer/footer';

describe('Form test',()=>{
  it('Footer exists',()=>{
    let app = shallow(<Footer/>);
    expect(app.find('footer').exists()).toBeTruthy();
  });

  it('renders footer correctly', ()=> {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
