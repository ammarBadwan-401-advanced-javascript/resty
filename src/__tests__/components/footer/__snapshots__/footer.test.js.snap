// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form test renders footer correctly 1`] = `
<footer>
   
  <div>
    Â© All right's reserved - Ammar 
  </div>
   
</footer>
`;
